packages/cli/src/databases/entities/user.ts

```diff
@@ -162,4 +162,9 @@ export class User extends WithTimestamps implements IUser {
 			return 'Unnamed Project';
 		}
 	}
+
+	toIUser(): IUser {
+		const { id, email, firstName, lastName } = this;
+		return { id, email, firstName, lastName };
```

Comment:
Not for this PR, but I think we should consider if we should hide user email also from these responses